src/main/java/com/coveros/selenified/element/Get.java

```diff
@@ -105,8 +105,7 @@ public String selectedOption() {
      */
     public String[] selectedOptions() {
         if (!isPresentSelect()) {
-            return null; // NOSONAR - returning an empty array could be confused
-            // with no options selected
+            return null; // returning an empty array could be confused with no options selected
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty array instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1168)